deploy/runtime/ui/Battery Lifetime.txt

```diff
@@ -971,14 +1026,36 @@ on_load{'Battery Lifetime'} = define() {
 	on_change{'batt_calendar_b'}();
 	on_change{'batt_calendar_c'}();
 	on_change{'batt_calendar_q0'}();
-
+	batt_chem = value('batt_chem');
+	if (batt_chem < 1) {
+		value('batt_life_model', 0);
+		enable('batt_life_model', false);
+	}
+	else {
+		on_change{'batt_life_model'}();
+		enable('batt_life_model', true);
+	}
```

Comment:
This enables/disables on the page load, but you need to add this check to `on_change{'batt_type'}` or `on_change{'batt_chem'}` so the page will update without having to reload the Battery System page. Right now changing batt_chem doesn't disable batt_life_model until you go to a different tab then return.

Reply:
Since `batt_chem`  isn't on the Lifetime page as it's on the Battery Model.txt, it doesn't seem to trigger off `batt_chem` being changed either. Is it supposed to work?

Reply:
Nevermind, adding code Battery Model.txt seems to disable batt_life_model correctly

Reply:
Yep that works like I would expect it to now.